SUI/SUI/BasketView.swift

```diff
@@ -0,0 +1,102 @@
+//  BasketView.swift
+//  SUI
+//  Created by angelina on 23.01.2023.
+
+import SwiftUI
+
+/// Экран корзины
+struct BasketView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let titleString = "Корзина"
+        static let typeString = "Вы выбрали тариф:"
+        static let priceString = "Цена:"
+        static let clearButtonTitle = "Очистить корзину"
+        static let emptyString = ""
+        static let buyButtonTitle = "Купить"
+        static let alertTitleString = "Поздравляем"
+        static let alertMessageString = "Вы оплатили тариф - "
+        static let titleBaselineOffsetNumber: CGFloat = 50
+        static let typeBaselineOffsetNumber: CGFloat = 30
+        static let fontSizeNumber: CGFloat = 25
+        static let minSpacerNumber: CGFloat = 30
+        static let widthButtonNumber: CGFloat = 200
+        static let heightButtonNumber: CGFloat = 50
+        static let cornerRadiusButtonNumber: CGFloat = 10
+        static let badgeNumber = 0
+    }
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        VStack {
+            Text(Constants.titleString)
+                .font(.largeTitle)
+                .bold()
+                .baselineOffset(Constants.titleBaselineOffsetNumber)
```

Comment:
```suggestion
            Text(Constants.titleString)
                .font(.largeTitle)
                .bold()
                .baselineOffset(Constants.titleBaselineOffsetNumber)
```
В вычисляемое свойство